app/data_model/list_store.py

```diff
@@ -12,24 +13,80 @@ def random_string(length):
     return ''.join(random.choice(ascii_letters) for _ in range(length))
 
 
+class ListRepresentation:
+    def __init__(self, list_items: List = None, context: Mapping = None):
+        if not list_items:
+            list_items = list()
+        if not context:
+            context = dict()
+
+        self.list_items = list_items
+        self.context = context
+
+    def __eq__(self, other):
+        if not isinstance(other, ListRepresentation):
+            return NotImplemented
+
+        return self.list_items == other.list_items and self.context == other.context
+
+    def remove_item_id(self, to_remove):
+        self.list_items.remove(to_remove)
```

Comment:
Currently, if I pass the primary persons list item id to the remove URL, it removes them. And then results in a 500 if I change the answer to `No` since the id no longer exists.
```sh
File "eq-survey-runner/app/data_model/list_store.py", line 33, in remove_item_id
    self.list_items.remove(to_remove)
ValueError: list.remove(x): x not in list
```

Reply:
Fixed.

Reply:
Fixed.